packages/gasket-plugin-https/lib/index.js

```diff
@@ -156,6 +156,52 @@ module.exports = {
           link: 'README.md#servers',
           parent: 'start',
           after: 'terminus'
+        }],
+        configurations: [{
+          name: 'http',
+          link: 'README.md#configuration',
+          description: 'HTTP port',
+          type: 'number'
+        }, {
+          name: 'https/http2',
```

Comment:
This is kinda confusing. Split these to separate entries with updated descriptions
```suggestion
          name: 'https',
```